js/webpack/libraries.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     // library
     'inject': ['./inject.js'],
     'web3': ['./web3.js'],
-    'parity': ['./parity.js']
+    'parity': ['./library.parity.js']
```

Comment:
Fully agree that it's not a for the library. @ngotchac and I discussed renaming it to `dev.parity.js`. What do you think?